### PR TITLE
Port to GTK3

### PR DESCRIPTION
**Pull request by sikmir**

### CODE REVIEW

**Comment by dfandrich** [comment]
Thanks for the patch! I'm not really up on GTK (which is why this hasn't been done yet) but is there any reason to keep GTK 2 support in 2019? Since this isn't a library there will be nothing else depending on it still being compatible with GTK 2.

File: gui.c [hunk 1]
```diff
@@ -100,7 +100,10 @@ GtkWidget *StripGPSButton;
 GtkWidget *PhotoListVBox;
 GtkWidget *PhotoListScroll;
 GtkWidget *PhotoList;
+#if GTK_CHECK_VERSION(2, 12, 0)
```
Comment (dfandrich) [inline gui.c]: This would be slightly cleaner as #if !GTK_CHECK_VERSION(2, 12, 0)

Comment (sikmir) [reply gui.c]: Fixed.

**Comment by sikmir** [comment]
> but is there any reason to keep GTK 2 support in 2019?

As for me, there is no reason. But... just in case, that's not a big deal. Anyway, it's up to you to skip GTK2 support.

**Comment by sikmir** [comment]
@dfandrich Can we land this PR?

File: Makefile [hunk 2]
```diff
@@ -13,11 +13,19 @@ GOBJS    = main-gui.o gui.o unixtime.o gpx-read.o correlate.o exif-gps.o latlong
 CFLAGS   = -Wall -O2
 CFLAGSINC := $(shell $(PKG_CONFIG) --cflags libxml-2.0 exiv2)
 # Add the gtk+ flags only when building the GUI
+ifeq (${GTK},3)
```
Comment (dfandrich) [inline Makefile]: This could be done without the GNU-specific ifeq by setting GTK=2 above and using it as gtk+-$(GTK).0 below.

Comment (sikmir) [reply Makefile]: Updated.

**Comment by dfandrich** [comment]
Thanks!

**Pull request merged**